src/test/resources/mocks/sscs-daily-list/sscsDailyList.json

```diff
@@ -39,41 +39,41 @@
                         "informant": [
```

Comment:
It is better to split up the test file into 2 to test party at hearing level and party at case level separated, so the old one can be easily removed in the future